### PR TITLE
snoop: Add version 3.0.0-alpha0116

### PR DESCRIPTION
**Pull request by gpailler**
Latest Snoop version. (marked as alpha but very stable).

### CODE REVIEW

File: bucket/snoop.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "version": "3.0.0-alpha.116",
+    "homepage": "https://github.com/snoopwpf/snoopwpf",
+    "license": "Ms-PL",
+    "description": "Snoop is the open source WPF spying utility created by Pete Blois when he was employed at Microsoft. It allows you to spy/browse the visual tree of a running application (without the need for a debugger) and change properties, view triggers, set breakpoints on property changes and many more things.",
```
Comment (Ash258) [inline bucket/snoop.json:5]:
```suggestion
    "version": "3.0.0-alpha0116",
    "description": "Spy/browse the visual tree of a running application (without the need for a debugger) and change properties, view triggers, set breakpoints on property changes and more.",
    "homepage": "https://github.com/snoopwpf/snoopwpf",
    "license": "MS-PL",
```

File: bucket/snoop.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    "url": "https://github.com/snoopwpf/snoopwpf/releases/download/v3.0.0-alpha0116/snoop.3.0.0-alpha0116.zip",
+    "hash": "f8cab3b1eaf3f623e1e86311ee69c49a46b7fa023c8b4962b762363609068847",
+    "shortcuts": [
+        [ "Snoop.exe", "Snoop" ]
+    ],
+    "checkver": {
+        "github": "https://github.com/snoopwpf/snoopwpf"
+    }
```
Comment (Ash258) [inline bucket/snoop.json:13]:
```suggestion
        "url": "https://github.com/snoopwpf/snoopwpf/releases",
        "regex": "tag/v([\\w.-]+)"
    },
    "autoupdate": {
        "url": "https://github.com/snoopwpf/snoopwpf/releases/download/v$version/snoop.$version.zip"
    }
```

File: bucket/snoop.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+    "version": "3.0.0-alpha.116",
+    "homepage": "https://github.com/snoopwpf/snoopwpf",
+    "license": "Ms-PL",
+    "description": "Snoop is the open source WPF spying utility created by Pete Blois when he was employed at Microsoft. It allows you to spy/browse the visual tree of a running application (without the need for a debugger) and change properties, view triggers, set breakpoints on property changes and many more things.",
+    "url": "https://github.com/snoopwpf/snoopwpf/releases/download/v3.0.0-alpha0116/snoop.3.0.0-alpha0116.zip",
+    "hash": "f8cab3b1eaf3f623e1e86311ee69c49a46b7fa023c8b4962b762363609068847",
+    "shortcuts": [
+        [ "Snoop.exe", "Snoop" ]
```
Comment (Ash258) [inline bucket/snoop.json:9]:
```suggestion
        [
            "Snoop.exe",
            "Snoop"
        ]
```

**Comment by gpailler** [comment]
I applied the requested changes. Thanks!

**Pull request merged**